.github/workflows/build.yml

```diff
@@ -0,0 +1,185 @@
+            sh <<'EOF'
+            # Setup the debian directory
+            mkdir -p ${SCRCPY_BUILD_DIR}/debian/DEBIAN
+            mkdir -p ${SCRCPY_BUILD_DIR}/debian/usr/local/bin
+            mkdir -p ${SCRCPY_BUILD_DIR}/debian/usr/local/share/scrcpy
+            mkdir -p ${SCRCPY_BUILD_DIR}/debian/usr/local/share/man/man1
+            mkdir -p ${SCRCPY_BUILD_DIR}/debian/usr/local/share/icons/hicolor/256x256/apps
+            mkdir -p ${SCRCPY_BUILD_DIR}/debian/usr/local/share/zsh/site-functions
+            mkdir -p ${SCRCPY_BUILD_DIR}/debian/usr/local/share/bash-completion/completions
+            mkdir -p ${SCRCPY_BUILD_DIR}/debian/usr/local/share/applications
+
+            # Create the control file
+            cat <<EOL >${SCRCPY_BUILD_DIR}/debian/DEBIAN/control
+          Package: scrcpy
+          Version: 2.2
+          Section: utils
+          Priority: optional
+          Architecture: amd64
+          Depends: ffmpeg, libsdl2-2.0-0, libusb-1.0-0
+          Suggests: adb
+          Maintainer: Scrcpy Developers <scrcpy-devs@github.com>
+          Description: Display and control your Android device
+           This application provides display and control of Android devices connected
+           on USB (or over TCP/IP). It does not require any root access.
+          EOL
+
+            cp -f ${SCRCPY_BUILD_DIR}/app/scrcpy ${SCRCPY_BUILD_DIR}/debian/usr/local/bin/
+            cp -f ${SCRCPY_BUILD_DIR}/server/scrcpy-server ${SCRCPY_BUILD_DIR}/debian/usr/local/share/scrcpy/scrcpy-server
+            cp -f app/scrcpy.1 ${SCRCPY_BUILD_DIR}/debian/usr/local/share/man/man1
+            cp -f app/data/icon.png ${SCRCPY_BUILD_DIR}/debian/usr/local/share/icons/hicolor/256x256/apps/scrcpy.png
+            cp -f app/data/zsh-completion/_scrcpy ${SCRCPY_BUILD_DIR}/debian/usr/local/share/zsh/site-functions
+            cp -f app/data/bash-completion/scrcpy ${SCRCPY_BUILD_DIR}/debian/usr/local/share/bash-completion/completions
+            cp -f app/data/scrcpy.desktop ${SCRCPY_BUILD_DIR}/debian/usr/local/share/applications
+            cp -f app/data/scrcpy-console.desktop ${SCRCPY_BUILD_DIR}/debian/usr/local/share/applications
+
+            dpkg-deb --build ${SCRCPY_BUILD_DIR}/debian ${SCRCPY_BUILD_DIR}/scrcpy_${DISTRIBUTION}_amd64.deb
+          EOF
+
+      - name: Fix Android SDK directory permissions
+        run: |
+          chown -R root:root ${ANDROID_HOME}
+
+      - name: Get Release ID
+        id: get_release
+        run: |
+          response=$(curl \
+            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            -H "Accept: application/vnd.github.v3+json" \
+            "https://api.github.com/repos/${{ github.repository }}/releases/tags/latest")
+          release_id=$(echo $response | jq .id)
+          echo "release_id=$release_id" >> $GITHUB_ENV
+        continue-on-error: true
+
+      - name: Create Release
+        if: env.release_id == ''
+        uses: ncipollo/release-action@v1
```

Comment:
This adds a dependency from a "random" user in the process, doesn't it? Ditto below.

Reply:
Github actions allow you to use community actions. The one you highlighted uses `ncipollo/release-action@v1` to create a release.

You can check the release here: https://github.com/hacksysteam/scrcpy/releases
The workflow file was designed to just create 1 release per push, we can make changes and adapt this to the project requirements. I just tested the bookworm deb file and it works, both server and app are included in the same deb file.

Reply:
I'll get it fixed.